impeller/entity/contents/radial_gradient_contents.h

```diff
@@ -54,6 +56,8 @@ class RadialGradientContents final : public ColorSourceContents {
   std::vector<Color> colors_;
   std::vector<Scalar> stops_;
   Entity::TileMode tile_mode_;
+  std::optional<Point> focus_;
+  Scalar focus_radius_;
```

Comment:
Best to zero out the scalars. This one and the one above. Also default the tilemode.